main.go

```diff
@@ -172,6 +172,18 @@ func fail(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+func findIosTargetPathComponent(target string, configuration string, cordovaVersion string) string {
+	if cordovaVersion != "" && cordovaVersion[0:1] >= "7" {
+		targetPlatform := "iphonesimulator"
+		if (target == "device") {
+			targetPlatform = "iphoneos"
+		}
+		return strings.Title(configuration) + "-" + targetPlatform
+	} else {
+		return target
+	}
+}
```

Comment:
The linter complains about the else branch because it has a single return statement so that else branch is just noise as you can simply move the return statement one level up and drop the else case.

But I would even go further because I like to flatten the if statements. I would do an early exit instead and do the logic afterwards:
```
func findIosTargetPathComponent(target string, configuration string, cordovaVersion string) string {
	if cordovaVersion == "" {
		return target
	}

	majorVersion, err := strconv.Atoi(cordovaVersion[0:1])
	if err != nil || majorVersion < 7 {
		return target
	}

	targetPlatform := "iphonesimulator"
	if target == "device" {
		targetPlatform = "iphoneos"
	}

	return strings.Title(configuration) + "-" + targetPlatform
}
```

Another question I have is that `strings.Title` is deprecated so we need to use something else. The comment recommends using `golang.org/x/text/cases` instead. But my question is that why do we need to make every words first letter uppercase? The available values for the configuration is `release` and `debug` with lowercase first letter.